kafka-java-auth/src/main/java/com/google/cloud/hosted/kafka/auth/GcpLoginCallbackHandler.java

```diff
@@ -104,8 +112,8 @@ public void handle(Callback[] callbacks) throws UnsupportedCallbackException, IO
     }
 
     for (Callback callback : callbacks) {
-      if (callback instanceof OAuthBearerTokenCallback) {
-        handleTokenCallback((OAuthBearerTokenCallback) callback);
+      if (callback instanceof OAuthBearerTokenCallback oAuthBearerTokenCallback) {
```

Comment:
```
The Java feature 'Type Patterns' is only available with source level 16 and above
```

Had to update `kafka-java-auth/pom.xml`:
 
```
<maven.compiler.release>16</maven.compiler.release>
```

Reply:
Thanks, added

Reply:
Actually, reverted as I think we want to be compatible with Maven 8